app/views/books/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Library Catalog</h1>
+
+<%= link_to "Add a new book", new_book_path %>
```

Comment:
Oh I just remembered! I think you wanted to try out i18n. If so you'll want to use it here and in the other views you've created